### PR TITLE
Mos branch/conditional1

### LINKED ISSUE #61: CONDITIONAL 1
CONDITIONAL 1:
echo "Installing prerequisites..."
sudo apt-get update && sudo apt-get install -y gnupg software-properties-common

Break this conditional into 3 parts:
1) sudo apt update
2) gnupg
3) software-properties-common

### PR DESCRIPTION
**Pull request by MonetCarriere**
The code first updates the package lists on the system using **sudo apt update** and displays a success or failure message based on the result. 

Then, it attempts to install the **gnupg** and **software-properties-common** packages using **sudo apt-get install -y**, which is gonna show a success or failure message depending on whether the installation is successful.

### CODE REVIEW

File: terraformInstall.sh [hunk 1]
```diff
@@ -1,34 +1,47 @@
 #!/bin/bash
 
 ## CONDITIONAL 1
-echo "Installing prerequisites..."
-sudo apt-get update && sudo apt-get install -y gnupg software-properties-common
+# Update package lists
+echo "Installing Prerequisites..."
+if sudo apt update; then
```
Comment (stephenlauck) [inline terraformInstall.sh]: sudo apt update is the resolution/remediation, not something you test. Try flipping it to where `apt-cache show PACKAGE` is the check and `sudo apt update` is the action in the else.

**Review by stephenlauck — CHANGES_REQUESTED** [review]

**Review by DevinBeu — APPROVED** [review]
this looks good to my baby brain

File: terraformInstall.sh (hunk 1 above)
Comment (MonetCarriere) [reply terraformInstall.sh]: I added the apt-cache show revision as you suggested.

File: terraformInstall.sh
Comment (Topher2014) [inline terraformInstall.sh:1]: The third conditional needs to be broken up into two: one to check if gnupg is installed, and another to check if software-properties-common is installed. Also, the "if" is not checking something, but is running an install command (back to Steve's comment about the 'resolution/remediation').

Comment (MonetCarriere) [reply terraformInstall.sh:1]: It should be good now.

**Review by Topher2014 — APPROVED** [review]

**Review by jlotto8 — APPROVED** [review]
👏

**Pull request merged**